spec.bs

```diff
@@ -1187,6 +1235,101 @@ To <dfn>report win</dfn> given a [=leading bid info=] |leadingBidInfo|, a [=stri
   1. TODO: Store |resultUrl| and |reportingBeaconMap| in the {{FencedFrameConfig}} as appropriate.
 </div>
 
+# K-anonymity # {#k-anonymity}
+  
+Two goals of this specification rely on applying k-anonymity thresholds:
+  
+  * To prevent cross-site leaks:  Inputs to event-level reporting functions, `reportWin()` and
+    `reportResult()`, only contain limited cross-site information. As described in
+    [[#privacy-considerations]], part of this limiting is done by
+    applying k-anonymity requirements to the ad URLs.
+  * To prevent microtargeting:  The browser applies k-anonymity requirements on the ad URLs to

```

Comment:
I think we shouldn't tout no microtargeting as a benefit. It's not part of the mission we're after

Reply:
I think the privacy goal is critical, but I think the microtargeting goal is still a goal, hence it's still listed in the explainer as one of the goals.